scripts/auto-cpufreq-runit

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-exec /usr/local/bin/auto-cpufreq --daemon
+# test for update
+if git -c advice.detachedHead=false pull -v https://github.com/AdnanHodzic/auto-cpufreq.git | grep -q -v '^\s*\(Updating\|Already up-to-date\|From\s\)\s'; then
+    exit
+else          
+    # install the update
+    echo "i" | sudo ~/auto-cpufreq/auto-cpufreq-installer && \
```

Comment:
A runit service is run as root (unless it is a user service, which are rare, not enabled by default, a bit difficult to set up and the least common way services are used in runit).

Using `sudo` doesn't make sense here, the service is already root. It also adds a dependency on `sudo`, which doesn't have to be installed. Some people also use `doas` or other things.

Where is `~`? You can't use things like `~` in service scripts without having a deep thought about where, how and by which user will the script be run. I have a suspicion that this script is really tailored to be a user service, because a lot of these things don't make sense otherwise. If that is the case, it should be clearly stated (for example in the line after shebang in a comment).

Why is `&& \` used everywhere? `set +o errexit` or something similar should be sufficient.